cmd/thanos/rule.go

```diff
@@ -173,6 +176,7 @@ func runRule(
 	s3AccessKey string,
 	s3SecretKey string,
 	s3Insecure bool,
+	s3SignatureV2 bool,
```

Comment:
ditto

Reply:
It is still not used ): 
I think you need to add this to `396: s3Config := &s3.Config{`

Reply:
Fixed.